doc/manual/src/release-notes/rl-next.md

```diff
@@ -0,0 +1,4 @@
+# Release X.Y (202?-??-??)
+
+- Add a `load-limit` setting to control builder parallelism. This has
+  also been backported to the 2.18 and later release branches.
```

Comment:
Release notes should now be done as separate files in `doc/manual/rl-next/`, they'll be concatenated when we do a release.